tests/function-controller/pkg/apirule/apirule.go

```diff
@@ -44,6 +44,10 @@ func New(name string, c shared.Container) *APIRule {
 	}
 }
 
+func (s APIRule) GetName() string {
```

Comment:
For the sake of consistency, I'd change the receiver name to `a` and use pointer receiver instead of value receiver.